src/components/Article.jsx

```diff
@@ -1 +1,27 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ article }) => {
+  const today = article.date;
+  const dateISOarticle = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {article.title}
```

Comment:
It would be great to destructure the article to get all the needed props of article